proposals/20220919-airflow_orchestration.md

```diff
@@ -19,13 +19,29 @@ Orchestrator
 We are moving Airflow from tfx/orchestration to tfx-addons. Native support for Airflow won't be provided in the near future.
 
 ## Project Implementation
-Move Airflow from tfx/orchestration to tfx-addons.
-
+1. Copy tfx/orchestration/airflow to tfx-addons/airflow.
+2. Mark Airflow as deprecated in tfx/ and indicate that support will be dropped in 1-2 releases.
+3. Update TFX tutorials on www.tensorflow.org to indicate deprecated and moving to TFXA
+ 
 ## Project Dependencies
```

Comment:
These aren't dependencies.  For comparison, see:

https://github.com/tensorflow/tfx-addons/blob/main/proposals/20210525-feast_example_gen.md#project-dependencies

https://github.com/tensorflow/tfx-addons/blob/main/proposals/20210707-xgboost_evaluator.md#project-dependencies

https://github.com/tensorflow/tfx-addons/blob/main/proposals/20220117-exit-handler-slack.md#project-dependencies

etc.